src/domain/event/occurrences/Occurrences.tsx

```diff
@@ -251,6 +268,27 @@ const Occurrences: React.FC<Props> = ({
               <OccurrenceGroupInfo occurrence={occurrence} />
             </div>
           </div>
+          <div className={styles.infoSection}>
+            <div>
+              <IconGlyphEuro />
+            </div>
+            <div>
+              <div className={styles.infoTitle}>
+                <div data-testid="event-price">
+                  {isFree && t('event:occurrenceList.eventIsFree')}
+                  {!isFree && price}
+                </div>
+              </div>
+              {priceDescription !== '' && (
+                <p data-testid="event-priceDescription">{priceDescription}</p>
+              )}
+              {priceInfoUrl !== '' && (
+                <p data-testid="event-priceInfoUrl">
+                  {createLink('', priceInfoUrl)}
+                </p>
+              )}
```

Comment:
After deleting some default values above, this can also be

`priceInfoUrl && (...`

Reply:
Changed.